rotkehlchen/tests/unit/test_globaldb_asset_updates.py

```diff
@@ -298,7 +305,7 @@ def test_parse_full_insert(assets_updater, text, expected_data, error_msg):
         assert error_msg in str(excinfo.value)
 
 
-def test_some_updates_are_malformed(assets_updater):
+def test_some_updates_are_malformed(assets_updater: AssetsUpdater) -> None:
```

Comment:
Hmmm (maybe also @LefterisJP's opinion will make sense here) we don't have typings for fixtures in other test. Why not to add them?

Reply:
Is easier if you do the introduction of types step by step than all in one big PR. I did those because are in the same file I'm changing and is related to my changes.